src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -38,4 +38,12 @@
     @ManyToOne
     @JoinColumn(name = "cart_id")
     private Cart cart;
+
+    public Product(@NotNull String name, @NotNull double price, String description, Group productGroup, Cart cart) {

```

Comment:
@NotNull pojawia Ci się przed polem name i price, więc tutaj w konstruktorze będzie zbędny.

Reply:
Poprawione

Reply:
Dodałem z powrotem bo myslałem, że mi to powoduje błąd w sonarze, ale to jednak nie to...